### PR TITLE
Add missing htsmsg_destroy calls

### PR DESCRIPTION
**Pull request by Jalle19**
This should fix some minor memory leaks.

### CODE REVIEW

File: addons/pvr.tvh/src/HTSPConnection.cpp [hunk 1]
```diff
@@ -436,6 +438,8 @@ void CHTSPConnection::SendAuth
   /* Auth denied */
   if (htsmsg_get_u32_or_default(msg, "noaccess", 0) != 0)
     throw new AuthException("Invalid username or password");
+  
+  htsmsg_destroy(msg);
```
Comment (adamsutton) [inline addons/pvr.tvh/src/HTSPConnection.cpp]: The above exception means this will still leak.

File: addons/pvr.tvh/src/Tvheadend.cpp [hunk 2]
```diff
@@ -417,6 +419,7 @@ PVR_ERROR CTvheadend::GetRecordingEdl
   }
   
   *num = idx;
+  htsmsg_destroy(m);
```
Comment (adamsutton) [inline addons/pvr.tvh/src/Tvheadend.cpp:423]: There are other exit points here, this will still leak.

**Comment by adamsutton** [comment]
@Jalle19 good spot, but there are some you've missed.

**Comment by Jalle19** [comment]
@adamsutton AFAICT everything is fixed now, could you take another look?

**Pull request merged**